README.md

```diff
@@ -1,5 +1,11 @@
 # Smart Contract init
-Service Will initialize Streamrs Smart Contracts
+Service Will initialize Streamr's Smart Contracts
+
+smart-contracts-init container is no longer part of streamr-docker-dev. Instead we use the docker-compose.yml file here to build the preloaded parity images streamr/open-ethereum-poa-mainchain-preload1 and streamr/open-ethereum-poa-sidechain-preload1.
+
+To build images:
+./preload_parity_images.sh
+
 
 Parity Default Private Key 
```

Comment:
Maybe a little bit of prose here about what all these things are (addresses generated by the preload scripts, common to all streamr-docker-dev setups that use dockers generated from this repository?)

Reply:
ok